pype/plugins/global/publish/integrate_new.py

```diff
@@ -64,6 +64,9 @@ class IntegrateAssetNew(pyblish.api.InstancePlugin):
                 "scene",
                 "vrayproxy",
                 "render",
+                "render.local",
+                "prerender",
+                "prerender.local",
```

Comment:
still not sure about this. It should be enough to integrate based on the main family, so `render` and `prerender`

Reply:
this way is more explicit as we need to avoid the .farm family suffix for each of render and prerender family

Reply:
or perhaps there is better solution based on definition main family and `family.suffix` exclusion?

Reply:
my point is that by the time the instance get's here It definitelly has render or pre-render family if it was already rendered. In case is wasn't rendered. It shoudn't have `render` family at all at this point because that is being taken care of on the farm. 

In other words, anything that has `render.farm` family can't have `render` at the same time. `render` is reserved for only finished renders.

Reply:
yeah that makes sense.. thanks ;)